sentry_sdk/integrations/celery/__init__.py

```diff
@@ -70,11 +68,6 @@ def __init__(
         exclude_beat_tasks=None,
     ):
         # type: (bool, bool, Optional[List[str]]) -> None
-        warnings.warn(
```

Comment:
@szokeasaurusrex this `propagate_traces` is for the Celery integration, the other top level one is the one we want to deprecate, not this one.

Reply:
@sl0thentr0py got it. So you are saying we should only partially revert the change, right?

Reply:
yes

Reply:
@cmanallen is the deprecation warning causing test failures in Sentry this one or the one we meant to deprecate?

Reply:
the linked issue says its the celery one.

Reply:
The celery issue is all AFAIK.  I've not tested beyond that point.